common/views/components/SearchResults/AsyncSearchResults.js

```diff
@@ -39,7 +39,7 @@ class AsyncSearchResults extends Component<Props, State> {
         {!(this.state.items && this.state.items.length > 0) && (
           <div
             data-component="ContentListItems"
-            className="async-content component-list-placeholder"
```

Comment:
are we completely sure we don't need/want this?

Reply:
The only mention of `async-content` is in `_async_content.scss`, which wasn't being imported. I suppose the appropriate thing would be to put a `TODO` in to meet that intention.